components/expand-collapse/expand-collapse-content.js

```diff
@@ -56,6 +61,16 @@ class ExpandCollapseContent extends LitElement {
 				overflow: visible;
 			}
 
+			.d2l-expand-collapse-content-container[data-state="collapsed"],
+			.d2l-expand-collapse-content-container[data-state="collapsing"] {
+				opacity: 0;
+			}
```

Comment:
Can you clarify why these are necessary, when the default is already 0?

Reply:
Good catch, I think you're right! I thought I needed this initially...